### ISSUE TITLE
Config for bookmarks

### ISSUE THREAD

**Issue by SilkieSabra** [OpenCollarTeam/OpenCollar#741]
The current version of bookmarks (8.2.1) does not allow public access to bookmarks and does allow both Trusted and Owner to Add, Remove, and Send the wearer to the bookmarked slurl.  
I'd like to see a config setting for this, allowing owners to
1) turn on/off Trusted access to sending to bookmarks
2) turn on/off Wearer access to sending to bookmarks
3) turn on/off Trusted access to adding/removing bookmarks
4) turn on/off Wearer access to adding/removing bookmarks
5) alternatively, turn off Trusted or Wearer ability to remove a bookmark an owner has set.

This would make bookmarks a more fun and more "secure" feature.

**Comment by mistressohm** [comment]
I agree, the Bookmarks feature is great as an escape hatch or 'approved
list of locations' for a submissive.  But, it's a little too open for those
who truly want to control their owned pets/slaves yet still maintain the
ability to send them to locations without going there themselves,

A bookmark to a cage or jail comes to mind - the Owner wants to maintain
this as a possible punishment, and if the Wearer can access it (or other
Bookmarks) the punishment value is nil.  Likewise, Trusted access could
mean 'accidental' jailing.

I'd add a possible 6th config setting, for one bookmark (one!) that is
automatically triggered to send the Wearer if they safeword.  This would
effectively park the Wearer in a safe environment, where the Owner can
check on them.

(Runaway removes all restrictions and the wearer is free to go anywhere
they like. So no bookmark is needed.)
